src/Microsoft.AspNetCore.SignalR/SignalRDependencyInjectionExtensions.cs

```diff
@@ -15,10 +18,49 @@ public static ISignalRBuilder AddSignalR(this IServiceCollection services)
                 .AddJsonProtocol();
         }
 
+        public static ISignalRBuilder AddGlobalHubOptions(this ISignalRBuilder signalRBuilder, Action<HubOptions> options)
```

Comment:
This should be an overload of AddSignalR. There should be no global options call.

Reply:
Makes sense